api.md

```diff
@@ -840,7 +846,159 @@ SELECT reorder_chunk('_timescaledb_internal._hyper_1_10_chunk', 'conditions_devi
 runs a reorder on the `_timescaledb_internal._hyper_1_10_chunk` chunk using the `conditions_device_id_time_idx` index.
 
 ---
+## Continuous Aggregates :community_function: [](continuous-aggregates)
+TimescaleDB allows users the ability to automatically recompute aggregates 
+at predefined intervals and materialize the results. This is suitable for 
+frequently used queries.
+
+*  [CREATE VIEW](#cagg_create_view)
+*  [ALTER VIEW](#cagg_alter_view)
+*  [REFRESH](#cagg_refresh_view)
+*  [DROP VIEW](#cagg_drop_view)
+
+## CREATE VIEW (Continuous Aggregate) [](cagg_create_view) 
+`CREATE VIEW` statement is used to create continuous aggregates.
+
+The syntax is:
+``` sql
+    CREATE VIEW <view_name> [ ( column_name [, ...] ) ] 
+    WITH ( timescaledb.continuous [, timescaledb.<option> = <value> ] )
+    AS
+    <select_query>
+    
+    <select_query> is of the form :
+    SELECT <grouping_exprs>, <aggregate_functions>
+    FROM <hypertable>
+    [WHERE ... ]
+    GROUP BY <time_bucket( <const_value>, <partition_col_of_hypertable> ),
+             [ optional grouping exprs>]
+    [HAVING ...]
+```
+
+#### Parameters
+|Name|Description|
+|---|---|
+| `<view_name>` | Name (optionally schema-qualified) of continuous aggregate view to be created.|
+| `<column_name>`| Optional list of names to be used for columns of the view. If not given, the column names are deduced from the query.|
+| `WITH` clause | This clause specifies [options](#create-view-with) for the continuous aggregate view.|
+| `<select_query>`| A `SELECT` query that uses the specified syntax. |
+
+#### `WITH` clause options [](create-view-with)
+|Name|Description|Type|Default|
+|---|---|---|---|
+|**Required**||
```

Comment:
Looks like this currently.  

![Screen Shot 2019-05-03 at 11 34 29 AM](https://user-images.githubusercontent.com/12288273/57148448-9f343980-6d97-11e9-955e-eb74df6482a6.png)

Need to add two `|` to **Required** line and **Optional** line or remove one on each to make it look complete.  I favor removing one for each visually.

![Screen Shot 2019-05-03 at 11 37 44 AM](https://user-images.githubusercontent.com/12288273/57148596-e4586b80-6d97-11e9-8661-519d9408ba4a.png)

Reply:
have only 1 | now.